hack/get_module_template_from_build_job.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+## This script requires the following env variables:
+# PR_NUMBER (e.g. 82)
+# COMMIT_STATUS_JSON
+# PROJECT_ROOT (e.g. "../")
```

Comment:
If one of these env vars is missing will there be a meaningful error message?

Reply:
Not yet, but I will validate them in next PR where I need to modify this script for post-main job.